src/pages/using-spark/guides/content-style-guidelines.mdx

```diff
@@ -0,0 +1,100 @@
+  - [Start Case](#start-case)
+- [Quotation Marks](#quotation-marks)
+- [Accessibility Guidelines](#accessibility-guidelines)
+
+## Capitalization
+
+Use for visual presentation of pieces of text to make it easy for users to read.
+
+### Start Case
+
+Use this case style to capitalize the first letter of each word, regardless of length or part of speech. When using start case, avoid using end punctuation other than question marks.
+
+#### Usage
+
+Use for:
+
+- Headers that are descriptive or functional
+- All subheads regardless of whether the primary header is title or sentence case
+- All buttons labels (Primary, Secondary and Tertiary Buttons)
+- Menu items
+- Chart and table headers
+- Referring to the name of a page
+
+<h4 class=" docs-b-h4 sprk-b-TypeDisplayFive sprk-u-mbm sprk-u-Measure ">
+  Examples:
+</h4>
+
+- Homeowners Insurance Info
+- How To Buy A Home
+- Lock Your Rate And Save
+- Check Your To Do List
+
+### Sentence Case
+
+Sentence case capitalizes the first word of the sentence and proper nouns.
+
+#### Usage
+
+Use for:
+
+- Emotional and Impactful headers
+- Special instances, where there may be two primary headers on a page
+- Body copy
+- Email subject lines and pre-headers
+- List items, use end punctuation only when the list item is a complete sentence
+
+<h4 class=" docs-b-h4 sprk-b-TypeDisplayFive sprk-u-mbm sprk-u-Measure ">
+  Examples
+</h4>
+
+- We’re here to help!
+- If you want the lowest payment possible, you might want to look at a 30-year fixed.
+- Lock your rate and save.
+- Interest rates are on the rise.
+- Welcome to Rocket Mortgage.
+
+> As a general rule, use **Start case** for **Headers**, use either **Start** or **Sentence case** for drop down options and labels, depending on the context.
+
+## Quotation Marks
+
+Use opening and closing quotes to show the start and endpoint of quoted content.
+
+#### Usage
+
+- For quoting someone’s words, use double quotes.
+- Don’t use quotation marks when citing the full name of an online article within an interface.
+
+<h4 class=" docs-b-h4 sprk-b-TypeDisplayFive sprk-u-mbm sprk-u-Measure ">
+  Examples
+</h4>
+
+**Do's**
+
+- “My life has changed after using Rocket Pro. My crops have returned, my acne is clear, and my mental health has improved.” - Dana S. California Insurance Agent
+
+**Dont's**
+
+- Sort by “Submitted” to see documents you’ve sent us.
+- TechCrunch says, 'Rocket Mortgage offers an unbeatable tool.'
+- You can get more information from our “Home Buying Checklist For First-Time Home Buyers.”
+
+> For more information on **Content Style Guidelines**, refer to the [W3C Style Guide (Web Content Manual of Style)](https://www.w3.org/Guide/manual-of-style/) for additional insights.
+
+## Accessibility Guidelines
+
+These guides are not all-inclusive in regard to accessibility considerations.
+
+> Refer to the [Web Content Accessibility Guidelines (WCAG 2.1, Level AA)](https://www.w3.org/TR/WCAG21/) for additional information.
```

Comment:
Just an opinion: This doesn't have to be in quote breakout, this can be within the the text above it.
Using these quote blocks are best used when giving *additional* insight.